st2common/st2common/persistence/keyvalue.py

```diff
@@ -107,6 +109,11 @@ def get_by_scope_and_name(cls, scope, name):
         :rtype: :class:`KeyValuePairDB` or ``None``
         """
         query_result = cls.impl.query(scope=scope, name=name)
+
+        if not query_result:
```

Comment:
Wonder how this change affects other code like datastore service, etc. which uses this function.

Also looking at the code below, we used to return ``None`` if value didn't exist so just need to make sure all the code which uses it has also been updated and it has no negative side affects.

Reply:
Yep, it's handled in the LookUp classes.